### ISSUE TITLE
[Feature] Documents rating on Frontend

### ISSUE THREAD

**Issue by polubis** [polubis/4markdown#285]
**DoD**

1. Follows this US: https://github.com/polubis/4markdown/issues/282

**Comment by polubis** [comment]
BE: https://github.com/polubis/greenonsoftware-api/pull/27

**Comment by polubis** [comment]
FE: https://github.com/polubis/4markdown/pull/312

**Comment by polubis** [comment]
Can be tested here: https://soft-pithivier-093469.netlify.app/

**Comment by MajaWielechowska** [comment]
1-11 💚
12 ❤️ nothing is delayed, you can spam different sounds non stop.
13-15 💚

--

I also noticed that in Education Zone creation date changed place. I don't know if that supposed to be, beacuse it wasn't mentioned in this Feature.


![Image](https://github.com/user-attachments/assets/6a1832f5-f952-4593-9952-800edd12078f)

**Comment by polubis** [comment]
Yes, it was required to move it to top. About "delay" stuff - I'll remove this requirement from user story - it's not needed currently. We'll add it if any problems will be spotted out with overusing such mechanism, but currently - it's impossible - the feature is required to be signed in.
